hyperspy/_signals/eels.py

```diff
@@ -154,6 +154,50 @@ def generate_subshells(self, include_pre_edges=False):
                                 '%s_%s' % (element, shell))
                             e_shells.append(subshell)
 
+    def get_edges_near_energy(self, energy, width=1):
```

Comment:
1 eV is fairly narrow, would 5 or 10 be a more sensible default width?

Reply:
Yes sure, will set the default to 10.